app/models/grda_warehouse/health_emergency/isolation.rb

```diff
@@ -6,6 +6,13 @@
 
 module GrdaWarehouse::HealthEmergency
   class Isolation < GrdaWarehouseBase
-  include HealthEmergency
+    include HealthEmergency
 
+    def status
+      return 'In Isolation' if start_date && end_date.blank?
+      return "In Isolation until #{end_date}" if start_date
```

Comment:
I remember discussing that we should list ended isolations... I don't remember where we came down... Should there be `"Was in Isolation until #{end_date}?"`

Reply:
Not sure yet, I think we can leave it as is, and if the date is in the past, then it should be clear they don't need to be there anymore.  We should ask these details on the call with Boston tomorrow.